.rubocop.yml

```
@@ -1,23 +1,43 @@
+require: 
+  - rt_rubocop_defaults
+  - rubocop-rails
 AllCops:
-  NewCops: enable
+  TargetRubyVersion: 2.5
```

Could change this to 2.7 as that's what's currently running in prod.

---

would also suggest to just bump this to 2.7, especially since I already made use of some 2.7 feature in #294 (numbered parameters, the`_1` thingies)

```suggestion
  TargetRubyVersion: 2.7
```